client/pk-console.c

```diff
@@ -1652,9 +1654,21 @@ main (int argc, char *argv[])
 				ctx,
 				NULL);
 
+	// default progress bar padding and bar size to old values of 30 and 25 respectively
+	int bar_padding = 30;
+	int bar_size = 25;
```

Comment:
These need to be `guint` rather than `int` and also declared at the top of the function in C99 style.